ecocodepython.properties

```diff
@@ -14,7 +14,13 @@ defaults.mavenArtifactId=ecocode-python-plugin
 1.2.1.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.2.1
 
 1.3.0.description=Updates rules tags�; adds rules (EC10, EC66, EC203, EC404)�; upgrade librairies to SonarQube 10.0.0�; clean-up plugins and dependencies
-1.3.0.sqVersions=[9.9,LATEST]
+1.3.0.sqVersions=[9.9,10.1]
```

Comment:
This range needs to end with the current version so that your current users on SQ 10.3 will be prompted to upgrade.

Reply:
ok. thank you. I'm going to do this

Reply:
done